components/Portfolio/Trading/Overview/EqTable/index.tsx

```diff
@@ -82,17 +82,15 @@ const EquityTable = styled(({ className, holdings, currentPortfolio, allFilledOr
                     <EqTableRow>
                         <EqTableCellLarge>
                             <Amount color="#21DD53">
-                                {balances.quote.eq(0)
-                                    ? '-'
-                                    : currentPortfolio === 0
-                                    ? equitySum
-                                    : toApproxCurrency(balances.totalMargin)}
```

Comment:
we still want to show - here dont we

Reply:
I had a feeling all the '-'s were to be removed eventually since in a real app we're likely to not want that, but have added them back.

Reply:
I think the '-'s are for when the account is disconnected